ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -798,7 +828,7 @@ private StoreFindToken findEntriesFromSegmentStartOffset(long initialSegmentStar
       }
     }
 
-    if (newTokenOffsetInJournal != StoreFindToken.Uninitialized_Offset) {
+    if (newTokenOffsetInJournal != null) {
```

Comment:
Discussed this offline - it is set just a few lines below but the line doesn't show up because it didn't need to be changed.